src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="Moyo Company Logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              class="is-active section"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="section"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="section"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="section"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="section"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="section"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="section"
+              href="#"
```

Comment:
The 'href' attribute currently uses a placeholder '#'. Replace it with an actual URL to ensure proper navigation.